README.md

```diff
@@ -187,6 +187,32 @@ inputs:
         name_format: "{var_name}"
     target_output_variable: forcing
 
+  danra_derived_forcings:
```

Comment:
That wasn't possible before but i have now made it so that we can add both `variables` and `derived_variables` from the same dataset so I have moved everything under the other input group, as you suggested :)